### PR TITLE
Cleanup kiso testing repo

### PR DESCRIPTION
**Pull request by DamienKayser**
Just make some clean-up everywhere in the project

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -46,32 +46,31 @@ pipeline
         {
             steps
             {
-                script
-                {
-                    echo "Run all the unittests"
-                    sh 'pipenv run invoke unittest'
-                    // TODO: call coverage report + save it
-                }
+                sh 'pipenv run pytest --junitxml=reports/testReport.xml --ignore tests/test_cc_pcan_can.py'
```
Comment (sebastianpfischer) [inline Jenkinsfile:49]: I do not think we need the pcan_can information here.

File: NOTICE.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ Thanks goes to these wonderful people (sorted by last name):
  * Daniel Buehler
  * Sebastian Fischer
  * Damien Kayser
+ * Sebastian Clerson
```
Comment (sebastianpfischer) [inline NOTICE.md:22]: I'm a bit confused about the visualization here. Why are these elements in green and look like they are not there yet. When I check https://raw.githubusercontent.com/eclipse/kiso-testing/master/NOTICE.md, there are already there

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]
What I should review is not clear, it is somehow messed up with elements that are already on master

**Comment by sebastianpfischer** [comment]
Please resolve the merging conflicts too. It would be nice if you could rebase it. Thanks

**Comment by sebastianpfischer** [comment]
ECA check is also not passing. You probably need to update your eclipse settings or your git settings.

File: Jenkinsfile (hunk 1 above)
Comment (DamienKayser) [reply Jenkinsfile:49]: this part is for another york item

File: NOTICE.md (hunk 2 above)
Comment (DamienKayser) [reply NOTICE.md:22]: I take the last master, but I think I have oti take the branch origin/feature/one-year-contrbution ?

Comment (DamienKayser) [reply NOTICE.md:22]: Maybe it comes from the pre-commit hook and trailing white spaces

**Comment by DamienKayser** [comment]
Version issue

**Pull request closed without merging**